docs/en/admins/03_Updating.md

```diff
@@ -0,0 +1,92 @@
+
+First things first: we recommend to create a backup before updating:
+
+```sh
+# Perform all commands below in your FreshRSS directory:
+cd /usr/share/FreshRSS
+
+tar -czvf FreshRSS-backup.tgz .
+```
+
+The update process depends on your installation type, see below:
+
+
+## Using the web admin panel
+
+Change to your installation at http://localhost/FreshRSS/p/i/?c=update and hit the "Check for new updates" button.
+
+If there is a new version you will be prompted again. 
+
+
+## Using git
+
+If you manage FreshRSS via command line, then installing and updating FreshRSS can be done via git:
+
+```sh
+# If your local user does not have write access, prefix all commands by sudo:
+sudo ...
+
+# Perform all commands below in your FreshRSS directory:
+cd /usr/share/FreshRSS
+
+# Use the development version of FreshRSS
+git checkout -b dev origin/dev
+
+# Check out a specific version of FreshRSS
+# See release names on https://github.com/FreshRSS/FreshRSS/releases
+# You will then need to manually change version
+# or checkout master or dev branch to get new versions
+git checkout 1.7.0
+
+# Verify what branch is used
+git branch
+
+# Check whether there is a new version of FreshRSS,
+# assuming you are on the /master or /dev branch
+git fetch --all
+git status
+
+# Discard manual changes (do a backup before)
+git reset --hard
+# Then re-delete the file forcing the setup wizard
+rm data/do-install.txt
+
+# Delete manual additions (do a backup before)
+git clean -f -d
+
+# Update to a newer version of FreshRSS,
+# assuming you are on the /master or /dev branch
+git pull
```

Comment:
Since a `git fetch` was already executed above all this does is `git merge`. Not sure if it matters. :-P

Reply:
I just moved that existing docu here but didn't change the text. 
I think this part is not meant to be executed in that exact order, but more a bunch of useful commands on how to get an (updated) FreshRSS version. I don't believe that someone uses git for installation without enough know-how to perform a pull?! 
I don't mind changing it, but then we would need several chapters (use a tagged version, use the development version ...) and I don't want to write a git beginners instruction.